packages/material-ui-lab/src/PickersDay/PickersDay.tsx

```diff
@@ -350,15 +350,17 @@ const PickersDay = React.forwardRef(function PickersDay<TDate>(
     );
   }
 
+  const ariaLabelProp = !props.children ? { 'aria-label': utils.format(day, 'fullDate') } : {};
+
   return (
     <PickersDayRoot
+      {...ariaLabelProp}
```

Comment:
Let's inline the prop value directly. The indirection makes this code harder to read than necessary. Changing `aria-label` directly also makes it harder to track (i.e. git-blame) changes.

Reply:
Fixed